widgetsnbextension/src/manager.js

```diff
@@ -426,16 +426,13 @@ WidgetManager.prototype._updateCellSnapshots = function(cell, index) {
         // Remove URL information, so only the b64 encoded data
         // exists, because that's what the notebook likes.
         const imageMimetype = 'image/png';
-        const imageDataUrl = canvas.toDataURL(imageMimetype);
+        const imageDataUrl = canvas[0].toDataURL(imageMimetype);
         const imageData = imageDataUrl.split(',').slice(-1)[0];
 
         // Create a mime bundle.
         const bundle = {};
         bundle[imageMimetype] = imageData;
```

Comment:
We probably don't need the `imageMimetype` since the html already has the `<img>` tag.